src/core/storage/files.ts

```diff
@@ -0,0 +1,5 @@
+export interface Files {
```

Comment:
Para evitar o conflito com o NodeJS:
```suggestion
export interface Archive {
```